README.md

```diff
@@ -83,19 +97,50 @@ app.use(
 );
 ```
 
+### Migrating from cookie-based authentication to session tokens
+
+Versions prior to v4 of this package used cookies to store session information for your app. However, internet browsers have been moving to block 3rd party cookies, which creates issues for embedded apps.
+
+If you have an app using this package, you can migrate to session tokens by performing a few steps:
```

Comment:
```suggestion
If you have an app using this package, you can migrate from cookie-based authentication to session tokens by performing a few steps:
```

Reply:
Thanks for the copy check, applying them all!